pkg/amdsev/report.go

```diff
@@ -143,7 +143,7 @@ type reportResponse struct {
 }
 
 // FetchRawSNPReport returns attestation report bytes.
-func FetchRawSNPReport(reportData string) ([]byte, error) {
+func FetchRawSNPReport(reportData []byte) ([]byte, error) {
```

Comment:
Since we're making this an exported package, a little more scrutiny :) What's the use of exporting a function for fetching the raw report? Is it just a debugging aid?

Reply:
I feel like it makes sense to have a way to fetch the raw report in general, not just for debugging purposes. This way callers can implement their own decode logic, e.g. to have a different internal representation.

Reply:
Sure